dandiapi/api/templates/api/mail/dandiset_unembargo_failed.html

```diff
@@ -0,0 +1,6 @@
+There was an error during the unembargo of dandiset
+<a href="{{ dandiset.ui_link }}">{{dandiset.identifier }}</a>. This has been reported to the developers, and will be
+investigated as soon as possible. We hope to have this resolved soon! Please avoid making any major changes to this
+dandiset in the meantime, as that may hinder or delay the process of resolving this issue.
+<br><br><br>
+<em>You are receiving this email because you are an owner of this dandiset.</em>
```

Comment:
This is good copy. We should add a note saying to reply to this email (assuming it is sent by info@dandiarchive.org) if you haven't heard from us in such-and-such amount of time.

Reply:
[93d9392](https://github.com/dandi/dandi-archive/pull/1965/commits/93d93927d86fd10858ae6f51f5304ff9f848b77b)